packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.tsx

```diff
@@ -1,13 +1,19 @@
 import React, { PropsWithChildren } from "react";
 import classNames from "classnames";
 import { Icon } from "@jobber/components/Icon";
+import { InternalChipButton } from "@jobber/components/Chips/InternalChipButton";
 import { useChildComponent } from "../../hooks";
 import styles from "../../Chip.css";
 
 export function ChipSuffix({ children, className }: ChipSuffixProps) {
-  let singleChild = useChildComponent(children, d => d.type === Icon);
+  let singleChild = useChildComponent(
+    children,
+    d => d.type === Icon || d.type == InternalChipButton,
```

Comment:
Adding support for the dismissible button. This is a bit too much of a 'tight-couple' for me to be comfortable, but since it's just checking a type and not actually using functionality I was okay with it.